stable/atlantis/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.repoConfig -}}
```

Comment:
this file should be called configmap-repo-config.yaml so that it
1. Matches the naming convention of the `secret-*.yaml` files
1. Can be differentiated from other future config maps